app/validators/uuidValidator.js

```diff
@@ -0,0 +1,12 @@
+const { param } = require('express-validator');
+const { validate } = require('uuid');
+
+async function isUUIDValid(id) {
+    if (!validate(id)) {
+        return Promise.reject('Id must be valid UUID');
```

Comment:
why promis? you can return boolean

Reply:
I think we need to return promise or throw error

Reply:
yes - it should be promise

Reply:
isn't returning boolean looks better?
`custom(v => validate(v).withMessage('Id must be valid UUID')`?